File: backend/chat-service/src/main/java/com/example/chatservice/chat/repository/ChatRoomMemberRepository.java
package com.example.chatservice.chat.repository;

import com.example.chatservice.chat.entity.ChatRoomMember;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;
import java.util.Optional;

@Repository
public interface ChatRoomMemberRepository extends JpaRepository<ChatRoomMember, Long>, ChatRoomMemberCustomRepository {

    Optional<ChatRoomMember> findById(Long id);
}

File: backend/chat-service/src/main/kotlin/com/example/chatservice/chat/repository/ChatRoomMemberRepository.kt
package com.example.chatservice.chat.repository

import com.example.chatservice.chat.entity.ChatRoomMember
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ChatRoomMemberRepository : JpaRepository<ChatRoomMember, Long>, ChatRoomMemberCustomRepository
